doc/variables.yaml

```diff
@@ -2253,8 +2253,8 @@ values:
       similar fashion as TZ environment variable. For hints, look in
       /usr/share/zoneinfo. e.g. US/Pacific, Europe/Zurich, etc.
     args:
-      - (timezone
-      - (format))
+      - (timezone)
```

Comment:
Ohh, I see what happened here. I think we should probably do `(timezone (format))` on one line, as the current scheme doesn't really account for nested args like this.

Reply:
Oh, my bad. Just fixed it 👍🏼